h5p-row.js

```diff
@@ -30,12 +30,31 @@ H5P.Row = (function (EventDispatcher) {
     var createHTML = function () {
       // Create wrapper
       wrapper = document.createElement('div');
+      wrapper.classList.add('h5p-row');
 
-      // For now we just print the data :-)
-      const pre = document.createElement('pre');
-      pre.innerText = JSON.stringify(params, null, 2);
+      for (var i = 0; i < params.columns.length; i++) {
+        var column = document.createElement('div');
```

Comment:
I think `const` and `let` is cooler than `var`, but I don't know what the kids today think.